Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -337,6 +342,22 @@ cancelRidePopUpConfig state =
         }
   in
     cancelRideconfig'
+  
+genderBannerConfig :: ST.HomeScreenState -> Banner.Config
+genderBannerConfig state = 
+  let 
+    config = Banner.config
+    config' = config
+      { 
+        backgroundColor = Color.lightMintGreen
+      , title = "Complete your profile for a personalised ride experience"
```

Comment:
@Akash2172 Added translations for the string

Reply:
@Akash2172 This translation is still missing